### DISCUSSION TITLE
Hardware Suggestions

### DISCUSSION THREAD

**Discussion by nep2ner** [immich-app/immich#2032] [General]
I've been running Immich on a raspberry pi as I really like the idea of it being energy efficient. However I see that [v1.48.0](https://github.com/immich-app/immich/releases/tag/v1.48.0) dropped support for ArmV7. Will this be a common thing? I'm not very experienced with running servers so I'd like to make it as easy and secure as possible for myself going forward. What hardware is most likely to remain supported throughout future Immich development?

**Comment by jrasm91** [comment]
I believe we do want to continue to support that platform, but sometimes we may need to temporarily drop support as we figure out how to use new libraries and tools, which may not have support for it out of the gate.

**Reply by nep2ner** [reply]
Do you know, if I just remove the machine learning container from docker compose, can I run v1.48.0+ on ArmV7? Will that mess anything up other than the search?

**Reply by jrasm91** [reply]
You can certainly run it without machine learning and typesense. The two things that will stop working are (1) /search and (2) /explore.

To disable there are some env variables you can set to false.

**Comment by nep2ner** [comment]
Is there a way for me to specify [v1.47.3](https://github.com/immich-app/immich/releases/tag/v1.47.3) in my docker-compose file? I can't seem to figure out how to run that version of the server, it keeps trying to run the incompatible v1.48.0.

**Reply by nep2ner** [reply]
Can I do this by changing the `:release` at the end of each container inside the docker-compose file to `:v1.47.3`?

**Reply by jrasm91** [reply]
That's how the image is tagged I believe. You can always search the container registry for available tags.

https://hub.docker.com/r/altran1502/immich-server/tags

**Reply by nep2ner** [reply]
Thank you! Huge help!
